frontend/src/scenes/web-analytics/tabs/marketing-analytics/frontend/components/MarketingAnalyticsFilters/MarketingAnalyticsFilters.tsx

```diff
@@ -3,23 +3,28 @@ import { DateFilter } from 'lib/components/DateFilter/DateFilter'
 import { ReloadAll } from '~/queries/nodes/DataNode/Reload'
 import { CompareFilter } from 'lib/components/CompareFilter/CompareFilter'
 import { marketingAnalyticsLogic } from '../../logic/marketingAnalyticsLogic'
+import { dataNodeCollectionLogic } from '~/queries/nodes/DataNode/dataNodeCollectionLogic'
+import { MARKETING_ANALYTICS_DATA_COLLECTION_NODE_ID } from '../../logic/marketingAnalyticsTilesLogic'
+import { BindLogic } from 'kea'
 
 export const MarketingAnalyticsFilters = (): JSX.Element => {
     const { compareFilter, dateFilter } = useValues(marketingAnalyticsLogic)
     const { setCompareFilter, setDates } = useActions(marketingAnalyticsLogic)
 
     return (
-        <div className="flex flex-col md:flex-row md:justify-between gap-2">
-            <ReloadAll />
-            <div className="flex flex-row gap-2 items-center">
-                <CompareFilter compareFilter={compareFilter} updateCompareFilter={setCompareFilter} />
-                <DateFilter
-                    allowTimePrecision
-                    dateFrom={dateFilter.dateFrom}
-                    dateTo={dateFilter.dateTo}
-                    onChange={setDates}
-                />
+        <BindLogic logic={dataNodeCollectionLogic} props={{ key: MARKETING_ANALYTICS_DATA_COLLECTION_NODE_ID }}>
```

Comment:
fyi: I think we tend to put those up above, like the other common ones are in `WebAnalyticsDashboard` but I also think this is fine and works, so commenting for the sake of information :)